lib/controller/index.js

```diff
@@ -1,5 +1,5 @@
 const program = require('caporal');
-const preferenceManager = require('../utils/preference-manager');
+const preferenceManager = require('@utils/preference-manager');
 
```

Comment:
If we use the globals technique, then
`global.__defaultPrefPath = path.join(__dirname, '../../../../default-prefs.json');`
statement can be placed here and removed from every where else.

Reply:
This would be a good idea, sir. We can also read a json into the global scope.